static/js/marker.js

```diff
@@ -202,4 +215,236 @@ var MarkerView = Backbone.View.extend({
             this.marker.opacity = INACCURATE_MARKER_OPACITY / app.groupsData[group].opacity;
         }
     }
+
 });
+
+var localization =
```

Comment:
I've used
git fetch origin
git fetch origin master
then committed
then pushed

shouldn't that make sure my clone is up to date and pushing my own commits?